bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/bridge/slip/SClogin.java

```diff
@@ -112,15 +111,11 @@ public void setResponse(short responseCommand, byte[] thisResponseData, boolean
                 int cfmStatus = responseData.getOneByteValue(0);
                 switch (cfmStatus) {
                     case 0:
-                        logger.info("{} bridge connection successfully established (login succeeded).",
-                                VeluxBindingConstants.BINDING_ID);
-                        logger.debug("setResponse(): returned status: The request was successful.");
+                        logger.debug("setResponse(): bridge connection successfully established (login succeeded).");
```

Comment:
Suggest to leave it on level _info_ as a debugging will be possible without changing the debuglevel by novice users.

Reply:
@gs4711 my thinking was twofold..
- There is (also) a `logger.info("Velux Bridge '{}' initialization completed (with {} scenes and {} actuators)."` at the end of `initializeScheduled()`, and we should avoid too many logger.info() saying (essentially) the same thing.
- The normal user does not really need a logger.info() in the 'normal' case when login **succeeded**, although they do need a logger.warn() if the login **failed** -- which is what we have.